wallet/wallet-cli-lib/src/commands/command_handler/mod.rs

```diff
@@ -627,6 +681,71 @@ where
                 })
             }
 
+            WalletCommand::AddStandaloneKey { address, label } => {
+                let (wallet, selected_account) = wallet_and_selected_acc(&mut self.wallet).await?;
+                wallet.add_standalone_address(selected_account, address, label).await?;
+
+                Ok(ConsoleCommand::SetStatus {
+                    status: self.repl_status().await?,
+                    print_message: "Success, the new address has been added to the account".into(),
+                })
+            }
+
+            WalletCommand::AddStandalonePrivateKey {
+                hex_private_key: private_key,
+                label,
+            } => {
+                let (wallet, selected_account) = wallet_and_selected_acc(&mut self.wallet).await?;
+                wallet.add_standalone_private_key(selected_account, private_key, label).await?;
+
+                Ok(ConsoleCommand::SetStatus {
+                    status: self.repl_status().await?,
+                    print_message: "Success, the new private key has been added to the account"
+                        .into(),
+                })
+            }
+
+            WalletCommand::AddStandaloneMultisig {
+                min_required_signatures,
+                public_keys,
+                label,
+            } => {
+                let (wallet, selected_account) = wallet_and_selected_acc(&mut self.wallet).await?;
+                let multisig_address = wallet
+                    .add_standalone_multisig(
+                        selected_account,
+                        min_required_signatures,
+                        public_keys,
+                        label,
+                    )
+                    .await?;
+
+                Ok(ConsoleCommand::SetStatus {
+                    status: self.repl_status().await?,
+                    print_message: format!("Success. The following new multisig address has been added to the account\n{multisig_address}")
+                })
+            }
+
+            WalletCommand::ListMultisigUtxo {
+                utxo_type,
+                with_locked,
+                utxo_states,
+            } => {
+                let (wallet, selected_account) = wallet_and_selected_acc(&mut self.wallet).await?;
+                let utxos = wallet
+                    .get_multisig_utxos(
+                        selected_account,
+                        utxo_type.to_wallet_types(),
+                        CliUtxoState::to_wallet_states(utxo_states),
+                        with_locked.to_wallet_type(),
+                    )
+                    .await
+                    .map(serde_json::Value::Array)?;
+                Ok(ConsoleCommand::Print(
+                    serde_json::to_string(&utxos).expect("ok"),
+                ))
+            }
```

Comment:
This is only showing atoms, not decimal value, in the output struct.

Reply:
changed to use RpcTxOuput from the Rpc wallet side, will change the format to toml in another PR?

Reply:
Yes, put changing the format to toml in another PR after the sim tests of the API server. I wanna see how it looks before we decide on it.

Reply:
Feel free to resolve this after you read the message.